### DISCUSSION TITLE
Dedicated LFO for vibrato

### DISCUSSION THREAD

**Discussion by o010b** [SynthstromAudible/DelugeFirmware#1395] [Ideas]
I find the Deluge could use more modulation sources like more LFOs and envelopes. 

One fairly easy way could be to free up LFO1 to have a dedicated LFO for the vibrato. It could take over the mod rate/depth knobs and still have same functionality. No functionality lost, no need for additional controls, but one more LFO available to create complex sounds while maintaining vibrato as before.

**Comment by m-m-adams** [comment]
Great idea!

**Comment by manysounds** [comment]
Yes please
